plotting_themes.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
this file should not be top-level, but imo belongs in `examples`
with the changes to the path detection that should not be an issue or?

Reply:
This was changed, so feel free to resolve.